scripts/eurostat/gdp/README.md

```diff
@@ -0,0 +1,56 @@
+# Importing Eurostat yearly GDP into Data Commons
+
+Author: fpernice-google
+
+## Table of Contents
+
+1. [About the Dataset](#about-the-dataset)
+1. [Import Procedure](#Import-Procedure)
+
+## About the Dataset
+
+### Download URL
+
+ZIP file is available for download from https://ec.europa.eu/eurostat/data/database.
+
+### Overview
+
+> Gross Domestic Product (GDP) measures the overall level of economic activity in a country or region. The Eurostat publishes various forms of economic data, including GDP, inflation, and employment. Specifically on GDP, it publishes data at various levels of government, and with different measurement styles. We import all Eurostat GDP data.
+
+> Data Format: All considered data is measured in one of the following forms:
+1. "MIO_EUR": "Million euro",
+2. "EUR_HAB":	"Euro per inhabitant",
+3. "MIO_NAC": "Million units of national currency",
+4. "MIO_PPS":	"Million purchasing power standards (PPS)", or
+5. "PPS_HAB":	"Purchasing power standard (PPS) per inhabitant".
+
+
+### Notes and Caveats
```

Comment:
Similar to oldham's comment--Eurostat often has lettered footnotes like c, e, etc. Which ones do you see? While ok for first pass, when we go through all the international data for cleanup, we should update the mMethods, such that e.g. (e) for Estimate will be appended to the current mMethod.

Reply:
Added note about this. Thanks!

Reply:
Added a note.